worker/src/RTC/TransportCongestionControlServer.cpp

```diff
@@ -221,6 +221,8 @@ namespace RTC
 	{
 		MS_TRACE();
 
+		this->transportCcFeedbackPacket->Finish();
```

Comment:
This is amazing because indeed we were NEVER calling this method (but just in the C++ test).